lib/Core/Site/Values/Content.php

```diff
@@ -143,4 +176,42 @@ private function buildField(array $properties = [])
         $this->fields[$field->fieldDefIdentifier] = $field;
         $this->fieldsById[$field->id] = $field;
     }
+
+    private function getLocations()
```

Comment:
Hi @slaci, you're right, I will make it public and add the parameter.
I also missed to mention it in the docblock and the condition to include only visible Locations, will have to fix that as well.